docs/modules/operation/pages/database-reports/database.adoc

```diff
@@ -28,6 +28,92 @@ The {page-component-title} Report Engine passes a JDBC connection to the {page-c
 
 For more details on how JasperReports works, please refer to the Jaspersoft Studio link:http://community.jaspersoft.com/documentation/tibco-jaspersoft-studio-user-guide/v610/getting-started-jaspersoft-studio[official documentation].
 
+[[ga-database-report-use-existing-report]]
+
+== Using existing reports
```

Comment:
IMO, lines 208 and 209 are far enough out of date that they could safely be removed.